packages/test-scenarios/__tests__/scenario-test.js

```diff
@@ -1,15 +1,74 @@
 const { Scenarios, Project } = require('scenario-tester');
-const { dirname } = require('path');
+const { dirname, join } = require('path');
 const { merge } = require('lodash');
+const { realpathSync } = require('fs-extra');
+const tmp = require('tmp');
 
 jest.setTimeout(500000);
 
+const rootTmpDir = createTmpDir();
+
+function createTmpDir() {
+  return realpathSync(tmp.dirSync({ unsafeCleanup: true }).name);
+}
+
 async function classic(project) {
   merge(project.files, {
-    'ember-cli-build.js': `'use strict';
+    tests: {
+      unit: {
+        'with-hooks-test.js': `import { module, test } from 'qunit';
+import { getTestMetadata } from '@ember/test-helpers';
```

Comment:
could these template strings be extracted into a fixture? it might make it easier to reason about the intent of this setup.

Reply:
Already doing that now in a separate branch.